README.md

```diff
@@ -71,13 +65,7 @@ server.
 Carol machines are created from a binary and a parameterization array. Most machines will have an empty parameterization for now so we make an empty POST request to t
 
 ``` sh
-curl -vv -XPOST "$carol_url/binaries/ddd205ed60d8b094f0c7b5b46c4d9d9a0fe36f14fd306572a5c6191c1fd0a46b"
-```
-
-This will return a response like:
-
-``` json
-{"id":"7f5389c82658b4792e8b32b91817300b5169458fb82fd1eab63f33f36e1decaf"}
+machine_id=$( cargo run -p carlo -- create --carol-url "${carol_url}" --binary-id "${binary_id}" )
```

Comment:
maybe we can have a quickstart section which just does this all in one command and runs it against `carol.computer`. Then a more broken down version showing each step. We'd need to add `carlo activate <json string>` to do it all in one go.

Reply:
Awesome ideas. I think #30 might be the next step in the direction of activation so you can fetch the list of endpoints.